src/lib.rs

```diff
@@ -120,24 +159,19 @@ impl TryFrom<&str> for Personnummer {
             date,
             serial,
             control,
-            divider,
-            coordination: (day > 31),
+            seperator,
+            coordination: (day / COORDINATION_NUMBER) >= 1,
```

Comment:
If we want to guard that this is actually +60 as samordningsnummer has to be, can't we jut do 

```suggestion
            coordination: day > COORDINATION_NUMBER,
```

Since we always add 60 the lowest day would be 61 and we don't have to divide the day and compare fractions.

Reply:
Yes of course! Not sure what I was thinking there.